### ISSUE TITLE
Nice repo！

### ISSUE THREAD

**Issue by Lyttonkeepfoing** [aimagelab/DefConvs_HTR#1]
Hey~ I'm a researcher of HTR. I'm really interested about how to generate the Gif that the visualization of deformable convolution on handwritten texts. It's a nice Gif！ Could you share the code ? Thanks a lot, it will help me!

**Comment by silviaCascianelli** [comment]
Thank you for appreciating our work!
You can find an example of how to obtain the images needed to create the Gif in the "visualization" folder: once you generate the sequence of images, you can use them in any Gif maker app.
Hope it helps!

**Comment by Lyttonkeepfoing** [comment]
That's cool ! It's really helpful~ By the way, are you still exploring the HTR task？I'm trying to make something interesting with the deformable paradigm that's the reason why I found your paper. And it is also one of our contribution on HTR. Our model based on Transformer. We have now made some achievements in the direction of the mask mechanism and performance on two datasets are SOTA. Are you interested in this? I look forward to discussing or collaborating with you if possible~
